python/semantic_kernel/connectors/ai/open_ai/services/azure_chat_completion.py

```diff
@@ -273,9 +273,9 @@ def from_dict(cls, settings: Dict[str, str]) -> "AzureChatCompletion":
             default_headers=settings.get("default_headers"),
         )
 
-    def get_request_settings_class(self) -> "AIRequestSettings":
+    def get_request_settings_class(self) -> "PromptExecutionSettings":
```

Comment:
this needs to be renamed!

Reply:
This doesn't seem to have any effect on the logic so I'm going to leave it as-is for now. I can rename it in a follow-up PR if needed.